Tools/wasm/README.md

```diff
@@ -12,7 +12,12 @@ Users and developers are encouraged to use the script
 `Tools/wasm/wasm_build.py`. The tool automates the build process and provides
 assistance with installation of SDKs.
 
-## wasm32-emscripten build
+* [Build emscripten](https://github.com/python/cpython/tree/main/Tools/wasm#wasm32-emscripten-build)
+* [Serve Python WASM](https://github.com/python/cpython/tree/main/Tools/wasm#hosting-python-wasm-builds)
+* [Build WASI-SDK](https://github.com/python/cpython/tree/main/Tools/wasm#wasi-wasm32-wasi)
+* [Running Python in WASI]((https://github.com/python/cpython/tree/main/Tools/wasm#running-python-wasi)
+
```

Comment:
We should either use relative anchor links so this works offline or rely on GitHub's automatic table of contents for all Markdown files.